strr-platform-web/app/interfaces/platform-application.ts

```diff
@@ -0,0 +1,55 @@
+export interface PlatAppAddress {
+  country: string
+  address: string
+  addressLineTwo: string
+  city: string
+  province: string
+  postalCode: string
+}
+
+export interface PlatAppParty {
+  firstName: string
+  lastName: string
+  middleName: string
+  phoneNumber: string
+  extension: string
+  faxNumber: string
+  emailAddress: string
+}
+
+export interface PlatAppRep extends PlatAppParty {
+  jobTitle: string
+}
+
+export interface PlatAppBusinessDetails {
+  legalName: string
+  homeJurisdiction: string
+  businessNumber: string
+  consumerProtectionBCLicenceNumber: string
+  noticeOfNonComplianceEmail: string
+  noticeOfNonComplianceOptionalEmail: string
+  takeDownRequestEmail: string
+  takeDownRequestOptionalEmail: string
+  mailingAddress: PlatAppAddress
+  registeredOfficeOrAttorneyForServiceDetails: {
+    attorneyName: string
+    mailingAddress: PlatAppAddress
+  }
+}
+
+export interface PlatAppPlatformDetails {
+  brands: PlatBrand[]
+  listingSize: ListingSize
+}
+
+export interface PlatAppRegistration {
+  registrationType: ApplicationType
+  completingParty: PlatAppParty
+  platformRepresentatives: PlatAppRep[]
+  businessDetails: PlatAppBusinessDetails
+  platformDetails: PlatAppPlatformDetails
+}
+
+export interface PlatApp {
```

Comment:
You guys using that as a prefix from the platform specific stuff?

Reply:
I think this is a prefix for the strr api mappings. It might make more sense to prefix these with 'StrrApi' or just 'Api'? What do you think Dietrich?

Reply:
Yes that works for me, I'm was trying to differentiate as the platform application in case we end up merging this with hosts/property managers